src/test/java/org/openrewrite/staticanalysis/RemoveEmptyJavaDocParametersTest.java

```diff
@@ -380,14 +419,6 @@ void singleLineThrowsNoSpace() {
               java(
                 """
                   class Test {
-                      /**@throws*/
-                      void method() throws IllegalStateException {
-                      }
-                  }
-                  """,
-                """
-                  class Test {
-                      /***/
```

Comment:
Similar to above: we no longer seem to assert anything meaningful here; especially not in relation to the test name. Should the test be restored or removed?